pulp_2to3_migration/tests/functional/util.py

```diff
@@ -48,3 +58,33 @@ def get_psql_smash_cmd(sql_statement):
         'psql', '-c', sql_statement,
         f'postgresql://{user}:{password}@{host}/{dbname}'
     )
+
+
+def set_pulp2_snapshot(name):
+    """
+    Roll out a specific pulp 2 snapshot
+
+    Args:
+        name(str): directory name  of the pulp 2 snapshot
+
+    """
+    smash_cfg = smash_config.get_config()
+    smash_cli_client = cli.Client(smash_cfg)
+    pulp2_mongodb_conf = utils.get_pulp_setting(smash_cli_client, "PULP2_MONGODB")
+    mongodb_host = pulp2_mongodb_conf['seeds'].split(':')[0]
+
+    pulp2_fs_setup_script_path = '/tmp/set_pulp2.sh'
```

Comment:
Is this file supposed to be created externally?

Reply:
This script needs to be on the machine where pulp runs. Doesn't matter where. /tmp/ was an easy location to pick, any suggestions are welcome.
I saw pulp_deb also copying their scripts into /tmp/, probably for the same reason.

Reply:
It's something that a test is expected to do themselves, right?

I just wanted to make sure none of the code here is supposed to be creating it.

Reply:
It is copied to the container by the CI script https://github.com/pulp/pulp-2to3-migration/pull/274/files#diff-de486dbe208dad5a436ac7b007fe0db87114bc32af1aa5693bd088ff52cac8f8R19. So the code in this specific module doesn't care about putting it in place, it just using it.

Reply:
A test it self is supposed to call  this `set_pulp2_snaphot` util. Not necessarily each test but the setup class/steps.

Reply:
gotcha

Reply:
Well, I don't have any ideas on how to do this more cleanly, and I'm kind of resigned to the fact that the migration infra is going to be a mess no matter how much time we spend on it, so I have no objection to merging.